sed/loader/flash/loader.py

```diff
@@ -178,6 +177,24 @@ def available_channels(self) -> List:
         available_channels.remove("pulseId")
         return available_channels
 
+    @property
+    def schema_from_config(self) -> set:
+        """Returns the set of all channels of the dataframe based on the config file
+
+        This is the complete list of channels which will be in the dataframe after creation.
+        """
+
+        schema = {"electronId", "trainId", "pulseId"}
+        if self._config["dataframe"]["split_sector_id_from_dld_time"]:
+            schema.add(self._config["dataframe"]["sector_id_column"])
```

Comment:
This can raise a key error if this entry is not defined. Use .get with default False.

Reply:
True, and shoudl be for both "split_sector_id_from_dld_time" and "sector_id_column", even though redundant as they are interdependent.

Reply:
perhaps we can just remove the split_sector_id_from_dld_time and use sector_id_column. If the column str exists, we can split the sector id. Otherwise we don't. So then there won't be a need for split_sector_id_from_dld_time bool

Reply:
I think this would be more difficult to understand the functionality then. It just requires a few more tests.